desc/compute/_profiles.py

```diff
@@ -1474,3 +1474,32 @@ def _current_r(params, transforms, profiles, data, **kwargs):
 def _current_rr(params, transforms, profiles, data, **kwargs):
     data["current_rr"] = 2 * jnp.pi / mu_0 * data["I_rr"]
     return data
+
+
+@register_compute_fun(
+    name="shear",
+    label="\\rho \\partial_{\\rho} \\iota / \\iota",
+    units="~",
+    units_long="None",
+    description="Normalized derivative of the rotational transform",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="r",
+    data=["0", "iota", "iota_r", "iota_rr", "rho"],
+)
+def _shear(params, transforms, profiles, data, **kwargs):
+    eps = 1e2 * jnp.finfo(data["iota"].dtype).eps
+    data["shear"] = cond(
+        jnp.all(jnp.abs(data["iota"])) < eps,
```

Comment:
This should be `jnp.all(jnp.abs(data["iota"]) < eps)`

Reply:
To make this consistent with the tokamak definition of shear, you should also put a minus sign in front.

For tokamaks, global shear = rho/q dq/drho and, q = 1/iota. Hence, global_shear = -rho/iota d iota/drho

Reply:
I don't think we want to follow the tokamak sign convention, since this is a stellarator code. When stellarator people talk about "positive shear" I take that to mean an increasing rotational transform.

Reply:
The definitions are the same for both tokamaks and stellarators. Increasing iota radially outwards would mean negative shear. Feel free to double-check this.

Reply:
Here is an example [from this paper](https://www.jstage.jst.go.jp/article/pfr/3/0/3_0_S1004/_pdf/-char/ja) where the stellarator shear is defined to be positive, and the equivalent tokamak shear is negative. The sign is a convention, and it seems like each community chooses to make their own machines have positive values. (Stellarators almost always have radially increasing iota.)

![image](https://github.com/PlasmaControl/DESC/assets/33005725/e523bbd3-cf07-47bc-a224-ef0509841ab2)

Reply:
This is a paper by a relatively small group, and they call tokamaks negative shear. This is the first time I have seen such a convention. Here are some [notes by Helander] (https://pure.mpg.de/rest/items/item_2146015/component/file_2146014/content). Please see the first line on page 24.
Here's a paper by [Hudson and Hegna](https://doi.org/10.1063/1.1622669) where the global shear is proportional to -iota_r. 

Note that negative shear is not the same as positive shear as far as stability is concerned. Negative shear almost always leads to a more stable equilibrium than positive shear. Therefore, I strongly suggest putting a negative sign in the definition.

Reply:
I concede